app/components/documents/document-card/edit-modal.js

```diff
@@ -11,13 +11,13 @@ export default class DocumentsDocumentCardEditModalComponent extends Component {
    */
 
   @tracked isUploadingReplacementFile = false;
-  @tracked isUploadingReplacementSourceFile = false;
-  @tracked isDeletingSourceFile = false;
+  @tracked isUploadingReplacementDerivedFile = false;
+  @tracked isDeletingDerivedFile = false;
 
   @tracked name;
-  @tracked uploadedSourceFile;
+  @tracked uploadedDerivedFile;
   @tracked replacementFile;
```

Comment:
Can we consistently use `source` and `derived` in all properties/methods of this component to avoid confusion?